readme.md

```diff
@@ -468,6 +468,7 @@
 - [Productivity](https://github.com/jyguyomarch/awesome-productivity#readme)
 - [Niche Job Boards](https://github.com/tramcar/awesome-job-boards#readme)
 - [Programming Interviews](https://github.com/MaximAbramchuck/awesome-interview-questions#readme)
+- [Engineering Team Principles](https://github.com/posquit0/awesome-engineering-team-principles#readme)
```

Comment:
> The entry in the Awesome list should:
* Include a short description about the project/theme of the list. **It should not describe the list itself.**

Reply:
I resolved the conflict and added the description.